src/axom/primal/operators/evaluate_integral.hpp

```diff
@@ -0,0 +1,176 @@
+// Copyright (c) 2017-2022, Lawrence Livermore National Security, LLC and
+// other Axom Project Developers. See the top-level LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+/*!
+ * \file evaluate_integral.hpp
+ *
+ * \brief Consists of methods that evaluate integrals over regions defined
+ * by Bezier curves, such as 2D area integrals and scalar/vector field line integrals
+ *
+ * Line integrals are computed with simple 1D Gaussian Quadrature using rules generated
+ * by MFEM. 2D area integrals computed with "Spectral Mesh-Free Quadrature for Planar 
+ * Regions Bounded by Rational Parametric Curves" by David Gunderman et al.
+ */
+
+#ifndef PRIMAL_EVAL_INTEGRAL_HPP_
+#define PRIMAL_EVAL_INTEGRAL_HPP_
+
+// Axom includes
+#include "axom/config.hpp"  // for compile-time configuration options
```

Comment:
Minor suggestion:  Please remove unnecessary/overly specific comments

```suggestion
#include "axom/config.hpp"
```